pyof/v0x04/asynchronous/packet_in.py

```diff
@@ -82,4 +82,4 @@ def __init__(self, xid=None, buffer_id=None, total_len=None, reason=None,
         self.table_id = table_id
         self.cookie = cookie
         self.match = match
-        self.data = data
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.